test/lib.bash

```diff
@@ -45,7 +45,55 @@ function run_e2e_tests {
     && return 1
 }
 
+function run_knative_serving_tests {
+  (
+  # Setup a temporary GOPATH to safely check out the repository without breaking other things.
+  local tmp_gopath
```

Comment:
`local tmp_gopath failed` to register multiple variables as local

Reply:
You lost me here. Didn't understand your comment.